lib/client-assets.php

```diff
@@ -206,6 +206,11 @@ function gutenberg_register_scripts_and_styles() {
 		),
 		'after'
 	);
+	wp_add_inline_script(
+		'wp-api-fetch',
+		'wp.apiFetch.use( wp.apiFetch.fetchAllMiddleware );',
```

Comment:
Should this be an inline script? I mean this feels like a mandatory middleware which should be always added as the last middleware before `raw` so we can always add it in js?